internal/commands/push/command.go

```diff
@@ -16,6 +16,17 @@ import (
 	"github.com/briandowns/spinner"
 )
 
+const (
+	flagLocalPath           = "local"
+	flagRemote              = "remote"
+	flagIncludeDependencies = "include-dependencies"
+	flagIncludeNodeModules  = "include-node-modules"
+	flagIncludePackageJSON  = "include-package-JSON"
```

Comment:
avoid capitalization in these flag names
```suggestion
	flagIncludePackageJSON  = "include-package-json"
```